src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
+    switch (actions[action].type) {
```

Comment:
You basically get the action from the loop, so you can access the type with only action.type and all the other instances that you used it.